Dockerfile build failure

I notice latest builds failing:

error[E0658]: use of unstable library feature 'arc_unwrap_or_clone'
   --> /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/delta_kernel-0.1.1/src/schema.rs:461:9
    |
461 |         Arc::unwrap_or_clone(schema).into()
    |         ^^^^^^^^^^^^^^^^^^^^
    |
    = note: see issue #93610 <https://github.com/rust-lang/rust/issues/93610> for more information
For more information about this error, try `rustc --explain E0658`.
error: could not compile `delta_kernel` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM rust:1.75 AS builder
+FROM rust:1.79 AS builder
 
 RUN mkdir /build
 WORKDIR /build

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 .DEFAULT_GOAL := help
 
 CARGO:=$(shell which cargo)
-APP=target/debug/placementd-web
+APP=target/debug/kafka-delta-ingest
 SOURCES=$(shell find . -type f -iname '*.rs')
 
 $(APP): Cargo.toml $(SOURCES)

